libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SmartStoreFullTextSearchTest.java

```diff
@@ -69,7 +68,7 @@
     private long eileenEvaId;
 
     @Override
-    protected String getPasscode() {
+    protected String getEncryptionKey() {
```

Comment:
Should we replace these empty strings to some random non-empty encryption key to better represent reality?

Reply:
We could. Maybe a different PR for making smartstore tests more realistic?

Reply:
Sounds good.